### PR TITLE
Modify Welcome msg in index.html

### PR DESCRIPTION
**Pull request by harry23potter**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -9,6 +9,6 @@
   </head>
   <body>
 	<script src="index.js"></script>
-  <h1>Welcome to Potionfy! Enjoy your stay.</h1>
+  <h1>Welcome to Potionfy! Enjoy your stay here.</h1>
```
Comment (csuryansh23) [inline index.html:12]: Can please remove 'Enjoy your stay here' and make remaining message in Red.

**Review by csuryansh23 — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (harry23potter) [reply index.html:12]: Sure, can do.

Comment (harry23potter) [reply index.html]: Its done. Please review.

**Comment by harry23potter** [comment]
New changes done.

**Review by csuryansh23 — COMMENTED** [review]
***

**Pull request merged**